uiTests/pages/kimchi_project/login.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import os
+import utils
+
+from selenium.common.exceptions import TimeoutException
+
+# locators by ID
+USERNAME = "username"
+PASSWORD = "password"
+LOGIN_BUTTON = "btn-login"
+LOGIN_BAR = "user-login"
+
+# environment variables
+ENV_USER = "KIMCHI_USERNAME"
+ENV_PASS = "KIMCHI_PASSWORD"
+ENV_PORT = "KIMCHI_PORT"
+ENV_HOST = "KIMCHI_HOST"
+
+
+class KimchiLoginPage():
```

Comment:
Login page is in fact hosted by Wok. I think we should get it away from Kimchi tests and move to Wok.

Reply:
That's a very good point, but also changes drastically how we handle dependencies.

Should I do the same thing as we do for tests in Kimchi? Add the path of the wok tree:

```
      # ../../../../../ refers to the project root
      # ../../../../ refers to wok directory
      # ../../../ refers to plugins directory
      OUTPUT=$(PYTHONPATH=../../../../../:../../../../:../../../ \
```